<launch>
	<!-- The metal detector frame -->
	<node pkg="fsr_husky_data_viewer" type="metal_detector_viewer_node" respawn="false" name="metal_detector_viewer" output="screen">
	   <param name="personal_mine_limiar" value="32000" />
	   <param name="tank_mine_limiar" value="150000" />
	</node>
	<!-- The mine mapping frame -->
	<node pkg="mine_mapping" type="mine_mapping_node" respawn="false" name="mine_mapping" output="screen">
	</node>
	<!-- The metal detector frame -->
	<node pkg="fsr_husky_convert_mines_rviz" type="fsr_husky_convert_mines_rviz_node" respawn="false" name="mines_to_rviz" output="screen">
	   <param name="tank_mine_size" value="0.3" />
	   <param name="tank_mine_height" value="0.5" />
	   <param name="personal_mine_size" value="0.1" />
	   <param name="personal_mine_height" value="0.1" />
	   <param name="mine_resolution" value="0.01" />
	</node>
	<!-- The metal detector frame -->
	<node pkg="fsr_husky_convert_mines_rviz" type="fsr_husky_mines_to_text_node" respawn="false" name="mines_to_text" output="screen">
	</node>
	<!-- The metal detector frame -->
	<node pkg="fsr_husky_convert_mines_rviz" type="fsr_husky_mine_sound_node" respawn="false" name="mines_to_sound" output="screen">
	</node>
</launch>
